collector/osv/Cargo.toml

```diff
@@ -28,6 +28,7 @@ serde_json = "1.0.68"
 tokio = { version = "1.0", features = ["full"] }
 guac = { workspace = true }
 packageurl = "0.3.0"
+url = "*"
```

Comment:
This smells funny. Intentional?

Reply:
I assumed it's mostly "I don't care, and trust someone in the workspace who has a strong opinion to influence it"